mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -679,7 +679,8 @@ def __init__(self, ho, name=''):
         super(ResolutionHOMediator, self).__init__(ho, name)
         ho.connect("valueChanged", self.value_change)
         self._precision = 3
-
+        self._ho.energy_hwobj = self.getObjectByRole("energy")
+        self._ho.dtox = self.getObjectByRole("dtox")
```

Comment:
You do need to have the correct configuration in the corresponding resolution.xml for that to work though